axelor-sale/src/main/resources/views/SaleOrder.xml

```diff
@@ -453,12 +453,17 @@
       <panel name="contentPanel" title="Content">
 
         <panel-related name="detailLinesPanel" title="Details Lines"
-          readonlyIf="clientPartner == null" colSpan="12" height="30" field="saleOrderLineList"
-          form-view="sale-order-line-form" grid-view="sale-order-line-grid" orderBy="sequence"
+          readonlyIf="clientPartner ==
+          null" colSpan="12" height="30"
+          field="saleOrderLineList" form-view="sale-order-line-form"
+          grid-view="sale-order-line-grid" orderBy="sequence"
           onChange="action-sale-order-method-on-line-change,supplychain-action-create-shipment-cost-line"
-          canNew="!(clientPartner == null || statusSelect == 2 || (statusSelect == 3 &amp;&amp; !orderBeingEdited) || statusSelect == 4)"
-          canRemove="!(clientPartner == null || statusSelect == 2 || (statusSelect == 3 &amp;&amp; !orderBeingEdited) || statusSelect == 4)"
-          canMove="true" if="!__config__.app.getApp('sale')?.getActivateMultiLevelSaleOrderLines()"/>
+          canNew="!(clientPartner == null || statusSelect == 2 || (statusSelect == 3 &amp;&amp; !orderBeingEdited)
+          || statusSelect == 4)"
+          canRemove="!(clientPartner == null || statusSelect == 2 || (statusSelect == 3
+          &amp;&amp; !orderBeingEdited) || statusSelect == 4)"
+          canMove="true"
+          if="!__config__.app.getApp('sale')?.getActivateMultiLevelSaleOrderLines() &amp;&amp; !__config__.app.getApp('sale')?.getIsEditableGridEnabled()"/>
```

Comment:
I don't see what changes needed to be made here, please revert all unnecessary newline changes